src/constants/endpoints.json

```diff
@@ -1,10 +1,12 @@
 {
   "test": {
     "globalSetting": "global_setting",
-    "contentData": "contents?content_category=Home&start=0&lang=english"
+    "contentData": "contents?content_category=Home&start=0&lang=english",
+    "login": "login"
```

Comment:
do we need to hardcode the query params ?

Reply:
It should not be , but I did for time being